src/Composer/Installers/FeatherBBInstaller.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace Composer\Installers;
+
+class FeatherBBInstaller extends BaseInstaller
+{
+    protected $locations = array(
```

Comment:
It's still actual locations? I see that your repo have another structure: https://github.com/featherbb/featherbb